src/__tests__/index.node.js

```diff
@@ -1,25 +1,56 @@
 /* eslint-env node */
 
 import test from 'tape-cup';
-import Plugin from '../plugin';
-import withI18n from '../hoc';
+import App from 'fusion-react';
+import React from 'react';
+import {getSimulator} from 'fusion-test-utils';
+import Plugin, {
+  I18nToken,
+  I18nLoaderToken,
+  HydrationStateToken,
+  createI18nLoader,
+  Translate,
+  withTranslations,
+} from '../index.js';
 
-test('plugin', t => {
-  const data = {test: 'hello', interpolated: 'hi ${value}'};
-
-  t.equals(typeof Plugin, 'function');
-
-  const I18n = Plugin({
-    loadTranslations: () => ({translations: data, locale: 'en-US'}),
-  });
-  const ctx = {headers: {'accept-language': 'en-US'}, memoized: new Map()};
-  t.equals(typeof I18n.of(ctx).translate, 'function');
+test('exports', t => {
```

Comment:
Cool!  Do we want to do this with all plugins?  We haven't been consistent so far.

Reply:
I kinda like having this, but idk

Reply:
I think it's a nice to have.   Useful for adding a minimal amount of friction to removing an export and making that major breaking change.